### PR TITLE
You decide to dance randomly

### PR DESCRIPTION
**Pull request by awakenedimagination**

### CODE REVIEW

**Comment by morosebose** [comment]
yo @casey-collab please review

File: english/dance-randomly/dance-randomly.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+You decide to dance randomly. It seems like a good idea at the time.
```
Comment (casey-collab) [inline english/dance-randomly/dance-randomly.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by awakenedimagination** [comment]
Added to amnesia page! Voila! :)

**Comment by morosebose** [comment]
@casey-collab please review

**Comment by casey-collab** [comment]
Looks great! :four_leaf_clover:

**Pull request merged**